styles.css

```diff
@@ -0,0 +1,104 @@
+.container {
+  background: #1c1a19;
+  display: flex;
+  flex-direction: column;
+  left: 500px;
+  top: 38px;
+  position: absolute;
+  height: 660px;
+  width: 375px;
+}
```

Comment:
- [ ] You are also still missing the background image which was requested on the previous code review. please make sure it shows correctly on the page.

This part:
![image](https://user-images.githubusercontent.com/99830261/192069232-6986bcc6-faf5-4ec4-b01e-47c3b713d45a.png)